robyn/__init__.py

```diff
@@ -200,6 +200,9 @@ def set_request_header(self, key: str, value: str) -> None:
     def set_response_header(self, key: str, value: str) -> None:
         self.response_headers.set(key, value)
 
+    def exclude_response_headers(self, exclude_paths: Optional[List[str]]):
+        self.response_headers.set_exclude_paths(exclude_paths)
```

Comment:
@VishnuSanal , should this be a public function?

Reply:
yep, why not? what if the user wants to enforce it to some pages themselves?

Reply:
Then we'd need to add documentation :)